linta-android/src/main/java/com/swvl/lint/utils/ColorUtils.kt

```diff
@@ -83,7 +83,7 @@ object ColorUtils {
      * format if an alpha component exists.
      */
     fun getBestColorFormat(hexColor: String): String {
-        val newColor = hexColor.toUpperCase(Locale.US)
+        val newColor = hexColor.uppercase(Locale.US)
```

Comment:
Same here. No need to set a specific locale anymore with this extension function.

```suggestion
        val newColor = hexColor.uppercase()
```